tests/utils/dockercli/volumelifecycle.go

```diff
@@ -242,3 +242,22 @@ func GetVolumeProperties(volumeName, hostName string) (string, error) {
 	cmd := dockercli.InspectVolume + volumeName + " --format ' {{index .Status.capacity.size}} {{index .Status.diskformat}} {{index .Status \"attached to VM\"}}' | sed -e 's/<no value>/detached/' "
 	return ssh.InvokeCommand(hostName, cmd)
 }
+
+// VerifyVDVSIsRunning -  finds out if vDVS is running. This util can be useful
+// in scenarios where vm is powered-on and user wants to find out if
+// vm is fully up to be able to run docker volume commands.
+func VerifyVDVSIsRunning(ip string) bool {
```

Comment:
volumelifecycle.go is not a better home to host this util .. Please move to appropriate file.